services/src/subscribers/subscribers.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Body, Controller, Post } from '@nestjs/common';
+
+import { AddSubscriberDto } from './dto/subscribers.dto';
+import { SubscriberService } from './subscribers.service';
+
+@Controller('subscribers')
+export class AddSubscribersController {
+  constructor(private readonly subscriberService: SubscriberService) {}
+  @Post('add')
```

Comment:
maybe `@Post()` is enough for api path, let http verb describe itself

call api

**post**,  /subscribers